File: problem_solving_for_beginners/array_problems/python/find_max.py
# Find max elements in array

arr = []
n = int(input("Enter the size of array: "))
max = 0

for i in range(n):
    i = int(input("Enter element of array: "))
    arr.append(i)
    
#write your code here
max = arr[0]
for i in arr:
    if i > max:
        max = i

print(f"Max = {max}")

File: problem_solving_for_beginners/array_problems/c-programming/find_max.c
#include<stdio.h>
/**
 * main - Find max elements in array
 *
 * Return: 0 Always (success)
 */
int main(void)
{
    int arr[10], n, i, max, cur, next;

    printf("Enter the size of array: ");
    scanf("%d", &n);

    for(i = 0; i < n; i++)
    {
        printf("Enter the number of element: ");
        scanf("%d", &arr[i]);
    }

    max = arr[0];
    for (i = 0; i < n; i++)
    {
        if (arr[i] > max)
            max = arr[i];
    }
    
    printf("Max = %d\n", max);
    return 0;
}